packages/metro-transform-plugins/src/addParamsToDefineCall.js

```diff
@@ -23,7 +23,7 @@ function addParamsToDefineCall(
 ): string {
   const index = code.lastIndexOf(')');
   const params = paramsToAdd.map(param =>
-    param !== undefined ? JSON.stringify(param) : 'undefined',
+    param != null ? JSON.stringify(param) : 'undefined',
```

Comment:
why the change? I think this was meant to explicitly handle `undefined` and not "`null` or `undefined`"

Reply:
I'm guessing the linter suggested it, but it'd be wrong in this case. 

`null` can be serialised to JSON, `undefined` can't, so the distinction was deliberate.
```suggestion
    param !== undefined ? JSON.stringify(param) : 'undefined',
```

Reply:
it was auto format by the linter yeah, not my change